visualization/package.json

```diff
@@ -78,12 +78,11 @@
 		"ajv": "^6.12.6",
 		"color-convert": "^2.0.1",
 		"d3-hierarchy": "^2.0.0",
+		"fast-deep-equal": "^3.1.3",
```

Comment:
Please use https://github.com/lukeed/dequal instead. It is safer and does a better comparison.

Reply:
[dequal/lite](https://github.com/lukeed/dequal#dequallite) is already enough for us